samples/caf_sensor_manager/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ / {
+	sensor_stub: sensor_stub {
+		compatible = "nordic,sensor-stub";
+		label = "SENSOR_STUB";
+		generator = "sensor_stub_gen";
+		status = "okay";
+	};
+
+	agg0: agg0 {
+		compatible = "caf,aggregator";
+		sensor_descr = "accel_sim_xyz";
+		buf_data_length = <240>;
+		sensor_data_size = <24>;
```

Comment:
note: this is not following prop naming conventions, should be e.g. `sensor-data-size` (no underscores)

Reply:
I will fix this in other PR. We didn't know about this convention when we were adding compatible for that.